client/lib/analytics/utils/get-tracking-prefs.ts

```diff
@@ -63,19 +70,31 @@ export default function getTrackingPrefs(): TrackingPrefs {
 	}
 
 	const cookies = cookie.parse( document.cookie );
+	const isCountryGdpr = isCountryInGdprZone( cookies.country_code );
+	const isCountryCcpa = isRegionInCcpaZone( cookies.country_code, cookies.region );
 
-	if (
-		! isCountryInGdprZone( cookies.country_code ) &&
-		! isRegionInCcpaZone( cookies.country_code, cookies.region )
-	) {
-		return {
-			ok: true,
-			buckets: allBucketsTrue,
-		};
+	if ( ! isCountryGdpr && ! isCountryCcpa ) {
+		return prefsAllowAll;
 	}
 
-	return parseTrackingPrefs(
+	const { ok, buckets } = parseTrackingPrefs(
 		cookies[ TRACKING_PREFS_COOKIE_V2 ],
-		cookies[ TRACKING_PREFS_COOKIE_V1 ]
+		cookies[ TRACKING_PREFS_COOKIE_V1 ],
+		// default tracking mechanism for GDPR is opt-in, for CCPA is opt-out:
+		isCountryGdpr ? prefsDisallowAll : prefsAllowAll
```

Comment:
Having a ternary inside of a function call just seems off to me for some reason. Would it be better to save the result of this ternary to a variable instead?

Reply:
Hmm, so to create the const, and then pass this const as param to the function? Makes sense, it will be more readable.